stumpy/gpu_stump.py

```diff
@@ -170,16 +170,16 @@ def _compute_and_update_PI_kernel(
             )
 
             QT_out[0] = QT_first[i]
-        if math.isinf(M_T[j]) or math.isinf(μ_Q[i]):
+        if math.isinf(μ_Q[j]) or math.isinf(M_T[i]):
```

Comment:
It might be a good idea to swap the name of variables `i` and `j` so as to be consistent with `stumpy/stump.py` where `i` is used for `μ_Q` (which is associated with `T_A`) and `j` for `M_T` (which is associated with `T_B`).

Reply:
Sure... if it works